src/System.ComponentModel.Annotations/tests/Helpers.cs

```diff
@@ -0,0 +1,86 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Xunit;
+
+namespace System.ComponentModel.DataAnnotations.Tests
+{
+    public static class Helpers
```

Comment:
Thanks - I'm away from most forms of electronic communication until September, so will take a look then